apps/docs/src/app/core/component-docs/grid-list/grid-list-docs.component.html

```diff
@@ -0,0 +1,142 @@
+</description>
+<component-example>
+    <fd-grid-list-delete-example></fd-grid-list-delete-example>
+</component-example>
+<code-example [exampleFiles]="gridListDeleteExample"></code-example>
+
+<fd-docs-section-title id="group" componentName="grid-list">Group</fd-docs-section-title>
+<description>
+    If grouped, a group header is displayed above all items which belong to the corresponding group. The group header is
+    not interactive. To create grid list group headers use fd-grid-list-group-header component.
+</description>
+<component-example>
+    <fd-grid-list-group-example></fd-grid-list-group-example>
+</component-example>
+<code-example [exampleFiles]="gridListGroupExample"></code-example>
+
+<fd-docs-section-title id="states" componentName="grid-list">States</fd-docs-section-title>
+<description>
+    To apply state, set <code>[state]</code> as <code>unread|error|locked|draft</code>.
+</description>
+<component-example>
+    <fd-grid-list-states-example></fd-grid-list-states-example>
+</component-example>
+<code-example [exampleFiles]="gridListStatesExample"></code-example>
+
+<fd-docs-section-title id="statuses" componentName="grid-list">Status</fd-docs-section-title>
+<description>
+    A status indicator is shown in front of the item it needs to show that an item needs attention. To add status
+    indicator, set <code>[status]</code> as <code>success|warning|error|neutral</code>.
+</description>
+<component-example>
+    <fd-grid-list-statuses-example></fd-grid-list-statuses-example>
+</component-example>
+<code-example [exampleFiles]="gridListStatusesExample"></code-example>
+
+<fd-docs-section-title id="more" componentName="grid-list">More button</fd-docs-section-title>
+<description> The More button loads more items to the front end if not all items have yet been loaded. </description>
+<component-example>
+    <fd-grid-list-more-example></fd-grid-list-more-example>
+</component-example>
+<code-example [exampleFiles]="gridListMoreExample"></code-example>
+
+<fd-docs-section-title id="footer" componentName="grid-list">Footer</fd-docs-section-title>
+<description> The footer can be used to display additional static information relating to the content. </description>
+<component-example>
+    <fd-grid-list-footer-example></fd-grid-list-footer-example>
+</component-example>
+<code-example [exampleFiles]="gridListFooterExample"></code-example>
+
+<fd-docs-section-title id="dnd" componentName="grid-list">Drag and Drop</fd-docs-section-title>
+<component-example>
+    <fd-grid-list-dnd-example></fd-grid-list-dnd-example>
+</component-example>
+<code-example [exampleFiles]="gridListDndExample"></code-example>
+
+<fd-docs-section-title id="layoutPattern" componentName="grid-list">Layout</fd-docs-section-title>
+<description> 
+    Defines the column layout in the format `XLn-Ln-Mn-Sn` where n is the number of columns and can be different for each size.
+    Default: XL4-L3-M2-S1
```

Comment:
Could you please show in documentation what type of columns a developer can use. `12,8,6...etc`